invenio/templates/deployments/web.yaml

```diff
@@ -131,6 +139,11 @@ spec:
         - mountPath: /opt/invenio/var/instance/data
           name: shared-volume
         {{- end }}
+        {{- range .Values.web.volumeMounts_extra }}
```

Comment:
I would prefer if all extra adds would be prefixed with `extra_` to keep it inline with `extra_env` and `extra_secrets`.